packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -782,16 +784,14 @@ test('price falls precipitously', async t => {
   // @ts-expect-error type confusion
   const m = await subscriptionTracker(t, metricsSub);
   await m.assertInitial(reserveInitialState(run.makeEmpty()));
-  await manualTimer.tick();
-  await assertDebtIs(debtAmount.value);
-
-  await manualTimer.tick();
+  await manualTimer.tick(); // t 0->1, p 2200->19180
   await assertDebtIs(debtAmount.value);
 
-  await manualTimer.tick();
+  await manualTimer.tick(); // t 1->2, p 19180->1650
   await assertDebtIs(debtAmount.value);
 
-  await manualTimer.tick();
+  await manualTimer.tick(); // t 2->3, p 1650->150, liquidates
+  await eventLoopIteration();
```

Comment:
Isn't this `eventLoopIteration();` unnecessary?

Reply:
correct, I'll remove it